### PR TITLE
Change Viewpoint sample update 

### PR DESCRIPTION
**Pull request by macca400**
update to Change Viewpoint desktop sample general fixes

### CODE REVIEW

**Comment by macca400** [comment]
@katydalton can you review please.

File: src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/MapView/ChangeViewpoint/ChangeViewpoint.xaml [hunk 1]
```diff
@@ -19,26 +19,29 @@
                 HorizontalAlignment="Right" VerticalAlignment="Top"
                 Margin="30" Padding="20" Width="375">
             <StackPanel>
-                <TextBlock Text="Select one of the button to runt the related Viewpoint code"
```
Comment (katydalton) [inline src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/MapView/ChangeViewpoint/ChangeViewpoint.xaml]: Can we get this block and buttons centered in the StackPanel?

File: src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/MapView/ChangeViewpoint/ChangeViewpoint.xaml.cs [hunk 2]
```diff
@@ -43,13 +43,15 @@ public ChangeViewpoint()
...
         {
             try
             {
+                // Return to initial viewpoint so Animation curve can be demonstrated clearly. 
+                await MyMapView.SetViewpointAsync(MyMapView.Map.InitialViewpoint);
                 var viewpoint = new Viewpoint(edinburghEnvelope);
-                //Animates the changing of the viewpoint giving a smooth transition from the old to the new view
-                MyMapView.SetViewpointAsync(viewpoint, System.TimeSpan.FromSeconds(5));
+                // Animates the changing of the viewpoint giving a smooth transition from the old to the new view.
+                MyMapView.SetViewpointAsync(viewpoint, System.TimeSpan.FromSeconds(10));
```
Comment (katydalton) [inline src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/MapView/ChangeViewpoint/ChangeViewpoint.xaml.cs]: Does this need an "await"?

**Comment by katydalton** [comment]
@macca400 A couple comments but all good. You can merge when ready.

**Pull request merged**